posts/2020-10-23-beta-20.0.0.12.adoc

```diff
@@ -416,7 +416,7 @@ Or you can add the Web Profile convenience feature to enable all of the Jakarta
 [#java15]
 == Support for Java SE 15
 
-Any official Java SE 15 release from link:https://adoptopenjdk.net[AdoptOpenJDK], OpenJDK, or Oracle works with Open Liberty. Java SE 15 is not a long-term supported release, with standard support scheduled to end in March 2021.
+Any official Java SE 15 release from link:https://adoptopenjdk.net/?variant=openjdk15&jvmVariant=openj9[AdoptOpenJDK], OpenJDK, or Oracle works with Open Liberty. Java SE 15 is not a long-term supported release, with standard support scheduled to end in March 2021.
```

Comment:
I agree with Andy. I also think we can link OpenJDK to the URL https://www.eclipse.org/openj9/ and Oracle to https://openjdk.java.net/groups/hotspot

Reply:
Could definitely reword it as OpenJDK doesn't produce any binaries one their own (afaik). Oracle could be linked to https://jdk.java.net/15/ for sure though.

Reply:
Added the preset link for JDK15/J9 and added a link to Oracle JDKs.